styles/go-plus-panel.less

```diff
@@ -106,4 +106,16 @@
     justify-content: center;
     text-align: center;
   }
+
+  .go-plus-panel-body {
+    font-family: Menlo, Consolas, 'DejaVu Sans Mono', monospace;
```

Comment:
That's what we used to do. People complained that they wanted a different font size. The default font (if you don't set a fontFamily) is not monosoace.